app/src/main/java/com/framgia/soundclound/util/StringUtil.java

```diff
@@ -0,0 +1,16 @@
+package com.framgia.soundclound.util;
+
+/**
+ * Created by Sony on 1/5/2018.
+ */
+
+public class StringUtil {
+    public static String  convertUrl(String url, String genre, int limit, int offSet) {
+        StringBuffer stringBuffer = new StringBuffer();
```

Comment:
WARNING: [InsufficientStringBufferDeclaration] StringBuffer constructor is initialized with size 16, but has at least 26 characters appended.
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#InsufficientStringBufferDeclaration